packages/gator-permissions-snap/src/permissions/contextValidation.ts

```diff
@@ -62,7 +62,12 @@ export function validateAndParseAmount(
  * @param startTime - The start time string to validate.
  * @returns Validation error message or undefined if valid.
  */
-export function validateStartTime(startTime: string): string | undefined {
+export function validateStartTime(
```

Comment:
Shouldn't the startTime in the _context_ be required? 

My understanding is that the startTime on the permission request object is optional, but if not provided, we coalesce to the current timestamp when creating the context object.

Edit: actually, looking at the `buildContext` functions we are _always_ defining a value, but the type specifies it as optional.

I think we should just mark it as required on the context object, and ensure that anything _after_ building the context requires the start time.